ui/lib/core/addon/components/string-list.js

```diff
@@ -31,6 +31,7 @@ import { removeFromArray } from 'vault/helpers/remove-from-array';
 
 export default class StringList extends Component {
   @tracked indicesWithComma = [];
+  @tracked inputList;
```

Comment:
it seemed like this should be tracked? It's originally set in the constructor, then updated in `addInput` and `removeInput`

Reply:
Since it's an ArrayProxy I'm not sure it really does 🤔 FWIW it seemed to work fine in my testing when I removed all the `addObject`s

Reply:
Okay, I can remove then!